### PR TITLE
review comment fix

### PR DESCRIPTION
**Pull request by purushottam08**
https://publicissapient.atlassian.net/browse/DTS-30141

### CODE REVIEW

File: customapi/src/test/java/com/publicissapient/kpidashboard/apis/common/service/KpiHelperServiceTest.java [hunk 1]
```diff
@@ -284,14 +283,14 @@ public void testFetchSprintCapacityDataFromDb() throws ApplicationException {
...
+		when(sprintRepository.findBySprintIDIn(any())).thenReturn(sprintDetailsList);
 
 		TreeAggregatorDetail treeAggregatorDetail = KPIHelperUtil.getTreeLeafNodesGroupedByFilter(kpiRequest, ahdList,
 				new ArrayList<>(), "hierarchyLevelOne", 5);
 		List<Node> leafNodeList = new ArrayList<>();
 		leafNodeList = KPIHelperUtil.getLeafNodes(treeAggregatorDetail.getRoot(), leafNodeList);
 
-		List<JiraIssue> resultList = kpiHelperService.fetchSprintCapacityDataFromDb(leafNodeList);
-		assertEquals(issueList.size(), resultList.size());
+		kpiHelperService.fetchSprintCapacityDataFromDb(leafNodeList);
```
Comment (Chittauri) [inline customapi/src/test/java/com/publicissapient/kpidashboard/apis/common/service/KpiHelperServiceTest.java:293]: add assert

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KpiDataHelper.java [hunk 2]
```diff
@@ -1061,4 +1063,25 @@ public static Map<String, String> processSprintIssues(List<JiraIssue> issues, St
 		return issueDateMap;
 	}
 
+	//function to calculate worklog of given issue with time frame
+	public static Double getWorkLogs(List<JiraHistoryChangeLog> worklogHistory, String sprintStartDate,
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KpiDataHelper.java:1067]: add java doc

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KPIExcelUtility.java [hunk 3]
```diff
@@ -636,25 +637,22 @@ public static void populateSprintPredictability(String sprint, Set<IssueDetails>
...
 				Map<String, String> storyDetails = new HashMap<>();
 				storyDetails.put(issue.getNumber(), checkEmptyURL(issue));
 				excelData.setStoryId(storyDetails);
 				excelData.setIssueDesc(checkEmptyName(issue));
 				setSquads(excelData, issue);
-				String daysLogged = "0.0";
 				String daysEstimated = "0.0";
-				if (issue.getTimeSpentInMinutes() != null && issue.getTimeSpentInMinutes() > 0) {
-					daysLogged = String.valueOf(roundingOff(Double.valueOf(issue.getTimeSpentInMinutes()) / 60));
+				if (loggedTimeIssueMap.get(issue.getNumber())!=null) {
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/util/KPIExcelUtility.java:653]: use get or default instead

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/model/LoggedTimePerIssue.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.publicissapient.kpidashboard.apis.model;
+
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/model/LoggedTimePerIssue.java:2]: copyright is missing

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/model/LoggedTimePerIssue.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class LoggedTimePerIssue {
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/model/LoggedTimePerIssue.java:10]: why this class is added, the comment is missing
author not added

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/scrum/service/SprintCapacityServiceImpl.java [hunk 6]
```diff
@@ -127,72 +140,66 @@ public KpiElement getKpiData(KpiRequest kpiRequest, KpiElement kpiElement,
...
+	 *            kpiRequest
 	 * @return {@code Map<String, Object>}
 	 */
 	@Override
 	public Map<String, Object> fetchKPIDataFromDb(List<Node> leafNodeList, String startDate, String endDate,
 			KpiRequest kpiRequest) {
-
-		Map<String, Object> resultListMap = new HashMap<>();
-		List<JiraIssue> sprintCapacityList = kpiHelperService.fetchSprintCapacityDataFromDb(leafNodeList);
+		Map<String, Object> resultListMap = kpiHelperService.fetchSprintCapacityDataFromDb(leafNodeList);
```
Comment (Chittauri) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/jira/scrum/service/SprintCapacityServiceImpl.java:155]: initialize resultListMap with new hashmap<>();

**Review by Chittauri — APPROVED** [review]

**Review by kunkambl — APPROVED** [review]

**Pull request merged**